charts/nri-bundle/values.yaml

```diff
@@ -0,0 +1,106 @@
+# Default values for nri-infrastructure.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# enables newrelic-infrastructure
+infrastructure:
+  enabled: true
+
+# enables nri-prometheus
+prometheus:
+  enabled: false
+
+# enables nri-metadata-injection
+webhook:
+  enabled: true
+
+# enables kube-state-metrics
+ksm:
+  enabled: false
+
+# enables nri-kube-events
+kubeEvents:
+  enabled: false
+
+# enables newrelic-logging
+logging:
+  enabled: false
+
+# enables newrelic-pixie
+newrelic-pixie:
+  enabled: false
+
+# enables pixie-chart
+pixie-chart:
+  enabled: false
+
+# enables newrelic-infra-operator
+newrelic-infra-operator:
+  enabled: false
+
+# enables newrelic-k8s-metrics-adapter
+metrics-adapter:
+  enabled: false
+
+# IMPORTANT: the following values are set as global so that they
+# can be shared by other newrelic product's charts
+#
+#   - Specify either the New Relic keys or the secret which
+#   contains them.
+#
+#   - Specify the Kubernetes cluster name.
+#     https://docs.newrelic.com/docs/kubernetes-monitoring-integration
+#
+# global:
+#   licenseKey:
```

Comment:
should we take care now of adding all info in the readme or in a follow up pr?

Reply:
It depends if the documentation for the common library is merged before this PR or not.

Reply:
Done in [d56dde6](https://github.com/newrelic/helm-charts/pull/795/commits/d56dde63a9cc0bbae5c7aa92a609d710cc5f485f)